package.json

```diff
@@ -6,6 +6,8 @@
     "packages/blocks/*",
     "packages/engine/apache-arrow-js-bundle",
     "packages/graph/migrations",
+    "packages/graph/hash_graph_clients",
```

Comment:
Preferable to just `packages/graph/clients`?

Reply:
Good idea. Do we still want to call the package `@hashintel/hash-graph-client`, though?